opencensus/stats/aggregation_data.py

```diff
@@ -221,15 +245,64 @@ def __init__(self, value):
         super(LastValueAggregationData, self).__init__(value)
         self._value = value
 
-    def add_sample(self, value):
+    def add_sample(self, value, timestamp, attachments):
```

Comment:
Same here.

Reply:
They are not used, but are needed to maintain the def signature. I changed these parameter to be optional.